backend/fms_core/serializers.py

```diff
@@ -226,7 +226,7 @@ class Meta:
         model = Taxon
         fields = "__all__"
         extra_fields = ("editable")
-    def get_editable(selft, obj):
+    def get_editable(self, obj):
         return not Individual.objects.filter(taxon_id=obj.id).exists()
```

Comment:
This would be cleaner to create a can_edit_taxon service in ../service/taxon.py to make sure we always have the same version of this test. This would include a small unit test also for the service.